app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java

```diff
@@ -37,28 +36,4 @@ public static void setTopBackgroundDrawable(@NonNull View view, @ColorInt int co
     GradientDrawable background = (GradientDrawable) view.getBackground();
     background.setColor(0xff000000 | colorRgb);
   }
-
-  /** Used to set a bottom rounded-rect background drawable with a data-bound color. */
-  @BindingAdapter("app:bottomRoundedRectDrawableWithColor")
-  public static void setBottomBackgroundDrawable(@NonNull View view, @ColorInt int colorRgb) {
-    view.setBackgroundResource(R.drawable.bottom_rounded_rect_background);
-    // The input color needs to have alpha channel prepended to it.
-    GradientDrawable background = (GradientDrawable) view.getBackground();
-    background.setColor(0xff000000 | colorRgb);
-  }
-
-  /** Used to set a rectangular background drawable with a data-bound color. */
-  @BindingAdapter("app:rectangleDrawableWithColor")
-  public static void setRectangleBackgroundDrawable(@NonNull View view, @ColorInt int colorRgb) {
-    view.setBackgroundResource(R.drawable.rectangle_background);
-    // The input color needs to have alpha channel prepended to it.
-    GradientDrawable background = (GradientDrawable) view.getBackground();
-    background.setColor(0xff000000 | colorRgb);
-  }
-
-  /** Used to set a resource background to a button. */
-  @BindingAdapter("android:button")
-  public static void setBackgroundResource(@NonNull Button button, @DrawableRes int resource) {
```

Comment:
But this seems to be used `android:button`. Should I add it?

![image](https://user-images.githubusercontent.com/78549195/150642582-5e57d4e0-4dfe-4220-b7b3-3cf22ecd4b4d.png)

Reply:
Do we ever actually use the databinding adapter, though? Note that it requires databinding syntax, e.g.:

```xml
android:button="@{...}"
```

If not, then this adapter isn't actually being used.